code/__HELPERS/pronouns.dm

```diff
@@ -57,6 +57,9 @@
 /datum/proc/p_es(temp_gender)
 	return "es"
 
+/datum/proc/p_themselves(temp_gender)
+	return "themselves"
```

Comment:
the default gender of datum implied by the other procs is "it", so this should be "itself"